examples/request-with-multiparts/request_with_multiparts.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on how Ballerina supports encoding and decoding multipart content in http requests along with nested parts.
```

Comment:
Edit: BBE on how Ballerina supports encoding and decoding multipart content in HTTP requests along with nested parts.